specs/windows/windows_optional_features.table

```diff
@@ -0,0 +1,13 @@
+table_name("windows_optional_features")
+description("Lists names and installation states of windows features. Maps to Win32_OptionalFeature WMI class.")
+schema([
+  Column("name", TEXT, "Name of the feature"),
+  Column("caption", TEXT, "Caption of feature in settings UI"),
+  Column("state", INTEGER, "Installation state value. 1 == Enabled, 2 == Disabled, 3 == Absent"),
+  Column("statename", TEXT, "Installation state name. 'Enabled','Disabled','Absent'"),
```

Comment:
nit: Perhaps just `value`? This column name seems a bit redundant, but that could just be me.

Reply:
you mean `state` and `statename`? It just depends how user wants to receive data.  programmatically, state is fine.  Interactively, I would want statename. I figured it's a lightweight table with maybe 200 entries, so not a big performance issue.